nginx-proxy/nginx.conf

```diff
@@ -0,0 +1,35 @@
+load_module modules/ngx_http_js_module.so;
+load_module modules/ngx_stream_js_module.so;
```

Comment:
that's not needed, right?

Reply:
This import is needed, njs will not work without it

Reply:
I looked at [the reference ](https://nginx.org/en/docs/njs/reference.html) and looks like stream module is only used for streams. That made me think it can be omitted.

Reply:
You are right. I've removed it